### DISCUSSION TITLE
Connect PnP Search Results to File and Media webpart

### DISCUSSION THREAD

**Discussion by P-IVI-T** [microsoft-search/pnp-modern-search#3341] [Q&A] [answered]
Hello! 

New to PnP modern search web part and I was wondering if anyone has successfully connected the PnP search results and file/media web parts? I was looking to search a document library and pipe it over to preview based on selection but this is not an available source connection in the drop-down list.

Thanks in advance 

PT

**Comment by kasperbolarsen** [comment, marked as answer]
Hi @P-IVI-T 
Just for your info, PnP Modern Search web parts are maintained by Community members and has no affiliation with Microsoft.
PnP Modern Search is, as far as I know, not intended to integrate with any of the web parts provided by MS. 

However, both the List and Card Layout supports previews out of the box. Is it something like this you are looking for?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/4ec1a5f8-a0b4-4158-a6f4-5590861649f9)

**Reply by P-IVI-T** [reply]
Thank you Kasper. Understood about the Microsoft affiliation. 

This does work for our needs. Any chance of making that thumbnail view resizable in the future? :)

Thanks again
